src/keen-slider.js

```diff
@@ -402,6 +402,9 @@ function KeenSlider(initialContainer, initialOptions = {}) {
   }
 
   function sliderGetSlidesPerView(option) {
+    if (option > length) {
```

Comment:
You could effectively change this function to return early, if the type of `option` is function and then have a clean return statement at the end

Reply:
I agree.